simpleSequential/floorDivision.py

```diff
@@ -9,3 +9,6 @@
 # Ex:
 #   floorDivision(18, 3) # return 6
 #   floorDivision(19, 4) # return 4
+
```

Comment:
Can you remove this extra line to keep the coding standard for the whole package?

Reply:
acknowledge.